docs/usage-as-end-user.md

```diff
@@ -223,5 +223,5 @@ Shoot clusters with Kubernetes v1.18 or less will use the in-tree `kubernetes.io
 ### Azure Accelerated Networking
 
 All worker machines of the cluster will be automatically configured to use [Azure Accelerated Networking](https://docs.microsoft.com/en-us/azure/virtual-network/create-vm-accelerated-networking-cli) if the prerequisites are fulfilled.
-The prerequisites are that the used machine type and operating system image version are compatible for Accelerated Networking.
+The prerequisites are that the cluster must be zoned and the used machine type and operating system image version are compatible for Accelerated Networking. `Availability Set` based shoot clusters will not be enabled for accelerated networking even if the machine type and operating system support it.
```

Comment:
Can you add more information here, i.e., the reason why we don't do respect the settings for AvSet clusters?

Reply:
done